script.lua

```diff
@@ -4112,6 +4114,8 @@ function swapPlayerPresenceColors(fromColor, toColor)
                         end
                         _ = obj.setState(originalState)
                     end
+                else
+                    broadcastToAll("Internal Error: Unknown object " .. suffix .. " in player bag.", Color.Red)
```

Comment:
this log message should be more clear that it's just printing the suffix of the object